src/blop/tests/test_agents.py

```diff
@@ -49,6 +66,7 @@ def test_agent(agent, RE, db):
 
 @pytest.mark.parametrize("agent", all_agents, indirect=True)
 def test_benchmark(agent, RE, db):
-    agent.db = db
-    per_iter_learn_kwargs_list = [{"acqf": "qr", "n": 32}, {"acqf": "qei", "n": 2, "iterations": 2}]
-    RE(agent.benchmark(output_dir="/tmp/blop", iterations=1, per_iter_learn_kwargs_list=per_iter_learn_kwargs_list))
+    if sys.version_info.minor > 9:
```

Comment:
Why is this needed?

Reply:
I see this in #100 as well actually and it had to do with numpy 2.0 being used, if that gets merged first then you can remove this check and the tests should pass.